src/main/java/com/exadel/movie_new/controller/MovieController.java

```diff
@@ -0,0 +1,37 @@
+package com.exadel.movie_new.controller;
+
+import com.exadel.movie_new.model.Movie;
+import com.exadel.movie_new.service.MovieService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+
+@RestController
+public class MovieController {
+    @Autowired

```

Comment:
Please use constructor injection instead of field injection. It will made your life easier while writing unit test. You don't need reflection to set the dependencies. You can just create mocks by yourself and inject them by simply calling the constructor.